config/sets/symfony/symfony5/symfony52.php

```diff
@@ -2,171 +2,23 @@
 
 declare(strict_types=1);
 
-use PHPStan\Type\ObjectType;
 use Rector\Config\RectorConfig;
-use Rector\Renaming\Rector\ClassConstFetch\RenameClassConstFetchRector;
-use Rector\Renaming\Rector\MethodCall\RenameMethodRector;
-use Rector\Renaming\Rector\PropertyFetch\RenamePropertyRector;
-use Rector\Renaming\ValueObject\MethodCallRename;
-use Rector\Renaming\ValueObject\RenameClassAndConstFetch;
-use Rector\Renaming\ValueObject\RenameProperty;
 use Rector\Symfony\Set\SymfonySetList;
-use Rector\Symfony\Symfony52\Rector\MethodCall\DefinitionAliasSetPrivateToSetPublicRector;
-use Rector\Symfony\Symfony52\Rector\MethodCall\FormBuilderSetDataMapperRector;
-use Rector\Symfony\Symfony52\Rector\MethodCall\ReflectionExtractorEnableMagicCallExtractorRector;
-use Rector\Symfony\Symfony52\Rector\MethodCall\ValidatorBuilderEnableAnnotationMappingRector;
-use Rector\Symfony\Symfony52\Rector\New_\PropertyAccessorCreationBooleanToFlagsRector;
-use Rector\Symfony\Symfony52\Rector\New_\PropertyPathMapperToDataMapperRector;
-use Rector\Symfony\Symfony52\Rector\StaticCall\BinaryFileResponseCreateToNewInstanceRector;
-use Rector\TypeDeclaration\Rector\ClassMethod\AddParamTypeDeclarationRector;
-use Rector\TypeDeclaration\ValueObject\AddParamTypeDeclaration;
 
 # https://github.com/symfony/symfony/blob/5.x/UPGRADE-5.2.md
 
 return static function (RectorConfig $rectorConfig): void {
-    $rectorConfig->sets([SymfonySetList::ANNOTATIONS_TO_ATTRIBUTES]);
+    // $rectorConfig->sets([SymfonySetList::ANNOTATIONS_TO_ATTRIBUTES]);
+
+    $rectorConfig->import(__DIR__ . '/symfony52/symfony52-dependency-injection.php');
+    $rectorConfig->import(__DIR__ . '/symfony52/symfony52-forms.php');
```

Comment:
I think the name should be `symfony52-form.php` instead of `symfony-forms.php`

```suggestion
    $rectorConfig->import(__DIR__ . '/symfony52/symfony52-form.php');
```

if name needs to follow component name https://github.com/symfony/form/blob/59b6c45ec432ff72b56c4ec5c68e0a107597623c/composer.json#L2

Reply:
I created PR:

- https://github.com/rectorphp/rector-symfony/pull/737

for it.

Reply:
Well done 👍